python/pyspark/serializers.py

```diff
@@ -199,6 +211,33 @@ def __repr__(self):
         return "ArrowSerializer"
 
 
+class ArrowPandasSerializer(ArrowSerializer):
+
+    def __init__(self):
+        super(ArrowPandasSerializer, self).__init__()
+
+    def dumps(self, series):
+        """
+        Make an ArrowRecordBatch from a Pandas Series and serialize
+        """
+        import pyarrow as pa
```

Comment:
Should we catch `ImportError`?

Reply:
Yeah, it would probably be best to handle it the same way as in `toPandas()`.

That got me thinking that it is a little weird to have an SQLConf "spark.sql.execution.arrow.enable" that is set for `toPandas()` but has no bearing with `pandas_udf`.  It doesn't need to since it is an explicit call but seems a little contradictory, what do you think?

Reply:
Ah, I see. Thanks for explaining it. Sure, I am okay with leaving it as is here.
I think we should catch and throw it with better messages in all cases (probably in the entry points) but probably let's talk about this later in another place.

Reply:
Ah, hm .. let me check the previous discussions and think about this a bit more.

Reply:
I am okay with leaving it as is here. I think we should catch and throw it with better messages in all cases later but let's talk about this in another place.